PhysicsTools/Heppy/python/physicsobjects/Electron.py

```diff
@@ -5,6 +5,25 @@
 import sys
 from math import exp
 
+effective_areas = dict(
+    ea03 = [
+        (1.000, 0.1440),
+        (1.479, 0.1562),
+        (2.000, 0.1032),
+        (2.200, 0.0859),
+        (2.300, 0.1116),
+        (2.400, 0.1321),
+        (2.500, 0.1654)        
+        ]
+)
+
+def get_effective_area(eta, area):
+    eta = abs(eta)
+    ea_defs = effective_areas[area]
+    for etamax, ea in ea_defs: 
+        if eta < etamax:
+            return ea 
+    return ea_defs[-1][1]
```

Comment:
also, I think that this and the above should be hidden from `import *` by having names starting with `_`, since they're not part of the public interface of the electron object and the name is unspecific enough that may clash with something else.